package.json

```diff
@@ -5,7 +5,10 @@
   "main": "index.js",
   "scripts": {
     "storybook": "start-storybook -p 9001 -c .storybook",
-    "build": "build-storybook -c .storybook -o ./docs"
+    "build": "build-storybook -c .storybook -o ./docs",
+    "prepare-release":
+      "git checkout master && git pull --rebase origin master && npm run docs && lerna updated",
```

Comment:
Doing the git stuff here seems weird to me. Could be annoying and cause problems.

Reply:
Yeah we can take it out. This is how "vx" handles stuff but I see no issue with removing it.

Reply:
I vote to kill it